CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ### Breaking Changes
 - Add request content length limit for the JSON-RPC API (5MB) [#5467](https://github.com/hyperledger/besu/pull/5467)
+- `min-block-occupancy-ratio` options is now ignored on PoS networks [#5491](https://github.com/hyperledger/besu/pull/5491)
```

Comment:
is this a breaking change?

Reply:
yes, but I think no one is really using this on a PoS network, because it makes no sense to force smaller blocks there.